doc/running.md

```diff
@@ -0,0 +1,68 @@
+# Running
+
+## Prerequisites
+
+This guide assumes you have the following tools:
+
+ * [Git](https://git-scm.com/download)
+ * [Cardano Node](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/getting-started/install.md)
+ * [Cardano DB Sync](../README.md)
+
+Create a working directory for your builds:
+
+```bash
+mkdir -p ~/src
```

Comment:
maybe we should point to `installing.md` as both here and there it says to `mkdir -p ~/src`